Algorithms.Tests/Strings/NaiveStringSearchTests.cs

```diff
@@ -0,0 +1,57 @@
+using Algorithms.Strings;
+using NUnit.Framework;
+using System.Linq;
+
+namespace Algorithms.Tests.Strings
+{
+   public static class NaiveStringSearchTests
+   {
+      [Test]
+      public static void ThreeMatchesFound_PassExpected()
+      {
+         // Arrange
+         var pattern = "ABB";
+         var content = "ABBBAAABBAABBBBAB";
+
+         // Act
+         var expectedOccurrences = new int[] {0, 6, 10};
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Remove the array type; it is redundant.](https://app.codacy.com/manual/siriak/C-Sharp/pullRequest?prid=4766632)